spec/message_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'message'
+
+describe Message do
+  let(:client) { double :client }
+  subject { described_class.new(client: client) }
+
+  describe "#send" do
+    it "should create a new txt message" do
+      expect(client).to receive_message_chain(:messages, :create)
+      subject.send
```

Comment:
if possible, it would be good to see if the message that was sent (the body) was as expected, and to test that your time method was working correctly.